pkg/log/log.go

```diff
@@ -197,44 +199,72 @@ type MyFormatter struct{}
 
 func (mf *MyFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 	var b *bytes.Buffer
+	var writeString string
 	if entry.Buffer != nil {
 		b = entry.Buffer
 	} else {
 		b = &bytes.Buffer{}
 	}
 	level := strings.ToUpper(entry.Level.String())
-	strList := strings.Split(entry.Caller.File, "/")
-	fileName := strList[len(strList)-1]
 	funcList := strings.Split(entry.Caller.Function, "/")
 	funcName := funcList[len(funcList)-1]
 	subIndex := strings.Index(funcName, ".")
 	if subIndex != -1 {
 		funcName = funcName[subIndex+1:]
 	}
 
-	b.WriteString(fmt.Sprintf("%s:[%s %s:#%d]  %s\n",
-		entry.Time.Format("2006-01-02 15:04:05 -0700"), level, fileName, entry.Caller.Line,
-		entry.Message))
+	// TODO: Once we migrate to ginkgo v2, we can extract test name as follows:
+	// report := CurrentSpecReport()
+	// testName := report.ContainerHierarchyTexts[0]
+	report := CurrentGinkgoTestDescription()
+	testName := strings.Split(report.FullTestText, " ")[0]
+	if testName != "" {
```

Comment:
can we have the format from "[{CancelClusterBackupShare}] [tests.dumpKubeConfigs:#3890]" to 
[CancelClusterBackupShare:tests.dumpKubeConfigs:#3890]

Reply:
LGTM